modules/client/src/addressList/internal/client/methods.ts

```diff
@@ -276,7 +276,9 @@ export class ClientAddressListMethods extends ClientCore
         return null;
       }
       // format in the metadata field
-      const metadataString = await this.ipfs.fetchString(addressListProposal.metadata);
+      const metadataUri = new URL(addressListProposal.metadata);
```

Comment:
The problem with this is that it's going to crash when the metadata does not have the correct format and throw a `GraphQLError`. For example, the metadata with the cid without the `ipfs://` will throw an error. I believe we should make a helper to get the metadata that checks this stuff and returns the metadata depending on the protocol, and if the protocol is not supported by the sdk return the same content that we have on the subgraph or null.

This is a first idea, let me know what do you think